ci/logging_tests_controller.yml

```diff
@@ -0,0 +1,141 @@
+    PATH: "{{ cifmw_path }}"
+  vars:
+    common_pod_test_id: "RHOSO-12752"
+    common_pod_status_str: "Running"
+    common_pod_nspace: openstack
+    common_pod_list:
+      - openstackclient
+  tasks:
+    - name: "Verify Running Pods"
+      ansible.builtin.import_role:
+        name: common
+
+
+- name: "Verify logging pods are running in openshift-operators-redhat"
+  hosts: controller
+  gather_facts: no
+  ignore_errors: true
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars:
+    common_pod_test_id: "RHOSO-12673"
+    common_pod_status_str: "Running"
+    common_pod_nspace: openshift-operators-redhat
+    common_pod_list:
+      - loki-operator-controller-manager
+
+  tasks:
+    - name: "Verify Pods running"
+      ansible.builtin.import_role:
+        name: common
+
+- name: "Verify logging pods are running in openshift-logging"
+  hosts: controller
+  gather_facts: no
+  ignore_errors: true
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars:
+    common_pod_test_id: "RHOSO-12676"
+    common_pod_status_str: "Running"
+    common_pod_nspace: openshift-logging
+    common_pod_list:
+      - cluster-logging-operator
+      - collector
+      - logging-loki-compactor
+      - logging-loki-distributor
+      #- logging-loki-gateway
+      - logging-loki-index-gateway
+      - logging-loki-ingester
+      - logging-loki-querier
+      - logging-loki-query-frontend
+      - logging-view-plugin
+
+  ### see JIRA LOG-5431 if pods not running
+  tasks:
+    - name: "Verify Pods running"
+      ansible.builtin.import_role:
+        name: common
+
+- name: "Verify logging pods are running in minio-dev"
+  hosts: controller
+  gather_facts: no
+  ignore_errors: true
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars:
+    common_pod_test_id: "RHOSO-12674"
+    common_pod_status_str: "Running"
+    common_pod_nspace: minio-dev
+    common_pod_list:
+      - minio
+
+  tasks:
+    - name: "Run pod running tests"
+      ansible.builtin.import_role:
+        name: common
+
+
+- name: "Verify logging pods have complete status in openstack"
+  hosts: controller
+  gather_facts: no
+  ignore_errors: true
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars:
+    common_pod_test_id: "RHOSO-12679"
+    common_pod_nspace: openstack
+    common_pod_status_str: "Completed"
+    common_pod_list:
+      - logging-edpm-deployment-openstack-edpm-ipam
+  tasks:
+    - name: "Run pods completed tests"
+      ansible.builtin.import_role:
+        name: common
+
+- name: "Verify the crds exist"
+  hosts: controller
+  gather_facts: no
+  ignore_errors: true
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars:
+    crd_test_id: "RHOSO-12670"
+    crd_list:
```

Comment:
These vars should be prefixed by the role name, since they belong to that role.
It's an ansible style thing that we haven't been enforcing so far, because it would need to be retroactively changed for old roles, but I would like to introduce it in new role that we're adding